### PR TITLE
add grobid tag

### PR DESCRIPTION
**Pull request by de-code**
related to https://github.com/elifesciences/sciencebeam-trainer-delft/pull/160
(this project should also then trigger the above)

### CODE REVIEW

File: Jenkinsfile.update-grobid [hunk 1]
```diff
@@ -0,0 +1,15 @@
+def grobidTag = params.grobid_tag
+
+elifeUpdatePipeline(
+    { _commit ->
+        sh "sed -i -e 's/LATEST_GROBID_TAG=.*/LATEST_GROBID_TAG=${grobidTag}/' .env"
+        sh "git add .env"
+    },
+    {
+        return "Updated grobid to ${grobidTag}"
```
Comment (giorgiosironi) [inline Jenkinsfile.update-grobid]: Maybe state it's only the latest grobid rather than all the other tags?

**Review by giorgiosironi — APPROVED** [review]

**Comment by giorgiosironi** [comment]
Do you need me to create the pipelines in https://alfred.elifesciences.org/job/dependencies/ or can @tayowonibi do this for you since he has the power?

**Comment by tayowonibi** [comment]
> Do you need me to create the pipelines in https://alfred.elifesciences.org/job/dependencies/ or can @tayowonibi do this for you since he has the power?

@de-code : let me know if you need me.

**Comment by de-code** [comment]
> > Do you need me to create the pipelines in https://alfred.elifesciences.org/job/dependencies/ or can @tayowonibi do this for you since he has the power?
> 
> @de-code : let me know if you need me.

@tayowonibi yes, that would be great if you could set up the pipeline for this and https://github.com/elifesciences/sciencebeam-trainer-delft/pull/160

**Pull request merged**